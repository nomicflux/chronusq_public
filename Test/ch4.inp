$geom
5
c 0 0.0000 0.0000 0.0000
h 0 0.6264 -0.6264 -0.6264
h 0 0.6264 0.6264 0.6264
h 0 -0.6264 0.6264 -0.6264
h 0 -0.6264 -0.6264 0.6264

$charge 0
$spin 1
$basis sto3g.gbs
$method hf
$print 3
$guess input
0 0 0 0 0 0 0 0 0 
0 0 0 0 0 0 0 0 0 
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0
