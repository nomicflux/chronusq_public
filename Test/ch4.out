Job started: Fri Apr 10 10:51:58 2015


Cartesian coordinates (bohr):
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                                      X              Y              Z
--------------------------------------------------------------------------------
       1       C       6     0.00000000     0.00000000     0.00000000
       2       H       1     1.18372445    -1.18372445    -1.18372445
       3       H       1     1.18372445     1.18372445     1.18372445
       4       H       1    -1.18372445     1.18372445    -1.18372445
       5       H       1    -1.18372445    -1.18372445     1.18372445
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Molecular Information:
       nAtoms =       5                 Charge =       0
   nElectrons =      10

Basis Function Information:
       nBasis =       9             nPrimitive =      27
      nSShell =       6                nPShell =       1
      nDShell =       0                nFShell =       0
      nGShell =       0

Cartesian Basis Functions:
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
          Center  lx  ly  lz          exponent       coefficient
--------------------------------------------------------------------------------
    1   S    1     0   0   0         71.616837          0.154329
                                     13.045096          0.535328
                                      3.530512          0.444635

    2   S    1     0   0   0          2.941249         -0.099967
                                      0.683483          0.399513
                                      0.222290          0.700115

    3   P    1     1   0   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   1   0          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957
                   0   0   1          2.941249          0.155916
                                      0.683483          0.607684
                                      0.222290          0.391957

    4   S    2     0   0   0          3.425251          0.154329
                                      0.623914          0.535328
                                      0.168855          0.444635

    5   S    3     0   0   0          3.425251          0.154329
                                      0.623914          0.535328
                                      0.168855          0.444635

    6   S    4     0   0   0          3.425251          0.154329
                                      0.623914          0.535328
                                      0.168855          0.444635

    7   S    5     0   0   0          3.425251          0.154329
                                      0.623914          0.535328
                                      0.168855          0.444635
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Atmoic Orbital Shell Pairs:
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
   Index    Center-1    Shell-1   L-1     *  Center-2    Shell-2   L-2
--------------------------------------------------------------------------------
       1           1          3     1     *         1          3     1
       2           1          3     1     *         1          1     0
       3           1          3     1     *         1          2     0
       4           1          3     1     *         2          4     0
       5           1          3     1     *         3          5     0
       6           1          3     1     *         4          6     0
       7           1          3     1     *         5          7     0
       8           1          1     0     *         1          1     0
       9           1          1     0     *         1          2     0
      10           1          1     0     *         2          4     0
      11           1          1     0     *         3          5     0
      12           1          1     0     *         4          6     0
      13           1          1     0     *         5          7     0
      14           1          2     0     *         1          2     0
      15           1          2     0     *         2          4     0
      16           1          2     0     *         3          5     0
      17           1          2     0     *         4          6     0
      18           1          2     0     *         5          7     0
      19           2          4     0     *         2          4     0
      20           2          4     0     *         3          5     0
      21           2          4     0     *         4          6     0
      22           2          4     0     *         5          7     0
      23           3          5     0     *         3          5     0
      24           3          5     0     *         4          6     0
      25           3          5     0     *         5          7     0
      26           4          6     0     *         4          6     0
      27           4          6     0     *         5          7     0
      28           5          7     0     *         5          7     0
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Single Slater Determinent Wave Function Information:
        nOccA =       5                  nVirA =       4
        nOccB =       5                  nVirB =       4
 Multiplicity =       1

Alpha Molecular Orbital Coefficients :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    2   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    3   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    6   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    7   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000

                   6              7              8              9
    1   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    2   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    3   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    6   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    7   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Alpha Density :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   0.0000000000
    2   0.0000000000   0.0000000000
    3   0.0000000000   0.0000000000   0.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    6   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    7   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000

                   6              7              8              9
    6   0.0000000000
    7   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for building the Coulomb matrix:0 seconds.

Alpha Coulomb Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   0.0000000000
    2   0.0000000000   0.0000000000
    3   0.0000000000   0.0000000000   0.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    6   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    7   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000

                   6              7              8              9
    6   0.0000000000
    7   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for building the Exchange matrix:0 seconds.

Alpha Exchange Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   0.0000000000
    2   0.0000000000   0.0000000000
    3   0.0000000000   0.0000000000   0.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    6   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    7   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000   0.0000000000

                   6              7              8              9
    6   0.0000000000
    7   0.0000000000   0.0000000000
    8   0.0000000000   0.0000000000   0.0000000000
    9   0.0000000000   0.0000000000   0.0000000000   0.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Overlap :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1   1.0000000000
    2   0.2483623971   1.0000000000
    3   0.0000000000   0.0000000000   1.0000000000
    4   0.0000000000   0.0000000000   0.0000000000   1.0000000000
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.0000000000
    6   0.0630131433   0.4936597908   0.2707775600  -0.2707775600  -0.2707775600
    7   0.0630131433   0.4936597908   0.2707775600   0.2707775600   0.2707775600
    8   0.0630131433   0.4936597908  -0.2707775600   0.2707775600  -0.2707775600
    9   0.0630131433   0.4936597908  -0.2707775600  -0.2707775600   0.2707775600

                   6              7              8              9
    6   1.0000000000
    7   0.1714356516   1.0000000000
    8   0.1714356516   0.1714356516   1.0000000000
    9   0.1714356516   0.1714356516   0.1714356516   1.0000000000
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Kinetic :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1  15.8911218138
    2  -0.0858899941   0.4722499926
    3   0.0000000000   0.0000000000   1.4777280906
    4   0.0000000000   0.0000000000   0.0000000000   1.4777280906
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   1.4777280906
    6  -0.0103428194   0.1085149700   0.1525865909  -0.1525865909  -0.1525865909
    7  -0.0103428194   0.1085149700   0.1525865909   0.1525865909   0.1525865909
    8  -0.0103428194   0.1085149700  -0.1525865909   0.1525865909  -0.1525865909
    9  -0.0103428194   0.1085149700  -0.1525865909  -0.1525865909   0.1525865909

                   6              7              8              9
    6   0.7600318800
    7  -0.0053057198   0.7600318800
    8  -0.0053057198  -0.0053057198   0.7600318800
    9  -0.0053057198  -0.0053057198  -0.0053057198   0.7600318800
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Potential :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1  35.6037029220
    2   4.6767743646   7.0845938195
    3   0.0000000000   0.0000000000   7.0427735524
    4   0.0000000000   0.0000000000   0.0000000000   7.0427735524
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000   7.0427735524
    6   1.2078402516   2.9817524616   1.5117745371  -1.5117745371  -1.5117745371
    7   1.2078402516   2.9817524616   1.5117745371   1.5117745371   1.5117745371
    8   1.2078402516   2.9817524616  -1.5117745371   1.5117745371  -1.5117745371
    9   1.2078402516   2.9817524616  -1.5117745371  -1.5117745371   1.5117745371

                   6              7              8              9
    6   4.9836568910
    7   0.9342689532   4.9836568910
    8   0.9342689532   0.9342689532   4.9836568910
    9   0.9342689532   0.9342689532   0.9342689532   4.9836568910
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

One Electron Integral :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1 -19.7125811082
    2  -4.7626643588  -6.6123438269
    3   0.0000000000   0.0000000000  -5.5650454618
    4   0.0000000000   0.0000000000   0.0000000000  -5.5650454618
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000  -5.5650454618
    6  -1.2181830711  -2.8732374915  -1.3591879462   1.3591879462   1.3591879462
    7  -1.2181830711  -2.8732374915  -1.3591879462  -1.3591879462  -1.3591879462
    8  -1.2181830711  -2.8732374915   1.3591879462  -1.3591879462   1.3591879462
    9  -1.2181830711  -2.8732374915   1.3591879462   1.3591879462  -1.3591879462

                   6              7              8              9
    6  -4.2236250110
    7  -0.9395746730  -4.2236250110
    8  -0.9395746730  -0.9395746730  -4.2236250110
    9  -0.9395746730  -0.9395746730  -0.9395746730  -4.2236250110
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

CPU time for one-electron integral:0 seconds.

Alpha Fock :
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
                   1              2              3              4              5
    1 -19.7125811082
    2  -4.7626643588  -6.6123438269
    3   0.0000000000   0.0000000000  -5.5650454618
    4   0.0000000000   0.0000000000   0.0000000000  -5.5650454618
    5   0.0000000000   0.0000000000   0.0000000000   0.0000000000  -5.5650454618
    6  -1.2181830711  -2.8732374915  -1.3591879462   1.3591879462   1.3591879462
    7  -1.2181830711  -2.8732374915  -1.3591879462  -1.3591879462  -1.3591879462
    8  -1.2181830711  -2.8732374915   1.3591879462  -1.3591879462   1.3591879462
    9  -1.2181830711  -2.8732374915   1.3591879462   1.3591879462  -1.3591879462

                   6              7              8              9
    6  -4.2236250110
    7  -0.9395746730  -4.2236250110
    8  -0.9395746730  -0.9395746730  -4.2236250110
    9  -0.9395746730  -0.9395746730  -0.9395746730  -4.2236250110
xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx

Energy Information:
            E(one electron) =    0.0000000000 a.u. 
            E(two electron) =    0.0000000000 a.u. 
       E(nuclear repulsion) =   13.4978430159 a.u. 
                   E(total) =   13.4978430159 a.u. 

Job finished: Fri Apr 10 10:51:58 2015

